libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -3,369 +3,380 @@ import { defineMessages } from 'react-intl'
 export const m = defineMessages({
   formTitle: {
     id: 'hi.application:title',
-    defaultMessage: 'Apply for health insurance',
+    defaultMessage: 'Umsókn um sjúkratryggingu',
     description: 'Apply for health insurance',
```

Comment:
While this is convenient to define the English translation in the description, that won't be used in any way inside the contentful extension to pre-populate the English translation field.

It's better to use the description to explain the message usage or where to find it for the people working on the translation later on in Contentful.

It's just quick information to keep in mind for future changes 👍

Reply:
Ok, will keep that in mind as we add new lines. We'll refactor the current as we go. :)